pyaerocom/io/read_eea_aqerep_base.py

```diff
@@ -282,8 +282,8 @@ def read_file(self, filename, var_name,
                 # UTF-8 decoding error
                 if suffix == '.gz':
                     os.remove(f_out.name)
-                    raise EEAv2FileError(
```

Comment:
This is wrong
EEAv2FileError is independent from the .gz suffix. pyaerocom should not delete files. Never
The os.remove just deletes a temp file in case a zipped file was read (which is not the case on PPI, but when I develop at home)

Reply:
Removed this indentation